src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +55,33 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
```

Comment:
Avoid explicit type casting by using a generic method in the `Injector` to prevent potential `ClassCastException`.